src/main/java/org/spongepowered/api/entity/EntityArchetype.java

```diff
@@ -65,6 +67,11 @@ static EntityArchetype of(EntityType type) {
      * Gets the raw {@link Entity} data that would be applied to the generated
      * entity. Note that this is a copied container.
      *
+     * <p>Note: While normally, an archetype does not contain position information,
```

Comment:
Does it make sense to create a new type extending EntityArchetype which essentially archetype+position?

Reply:
I was debating about it. Something like SchematicEntity? But that'd be a little enterprisey. It's also possible to grab entities from the ArchetypeVolume when creating the archetype volume through `Extent`, so that they're not limited to just Schematics (I went through to update implementation and found that if it's easier to basically keep pre-built plugins working with the same API and including entities by default.

So far, from my implementation in progress, I've been able to avoid modifying the CopyPasta plugin entirely, using the same API without modification.